sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/ImmunizationService.java

```diff
@@ -0,0 +1,55 @@
+package org.sormas.e2etests.services;
+
+import com.github.javafaker.Faker;
+import com.google.inject.Inject;
+import java.time.LocalDate;
+import java.util.UUID;
+import org.sormas.e2etests.enums.CommunityValues;
+import org.sormas.e2etests.enums.DistrictsValues;
+import org.sormas.e2etests.enums.RegionsValues;
+import org.sormas.e2etests.enums.immunizations.ImmunizationManagementStatusValues;
+import org.sormas.e2etests.enums.immunizations.StatusValues;
+import org.sormas.e2etests.pojo.web.Immunization;
+
+public class ImmunizationService {
+
+  private final Faker faker;
+
+  @Inject
+  public ImmunizationService(Faker faker) {
+    this.faker = faker;
+  }
+
+  public Immunization buildGeneratedImmunization() {
+    String timestamp = String.valueOf(System.currentTimeMillis());
+    return Immunization.builder()
+        .dateOfReport(LocalDate.now().minusDays(1))
+        .externalId(UUID.randomUUID().toString())
+        .disease("COVID-19")
```

Comment:
change this one with: DiseasesValues.getRandomDiseaseCaption()  so we can use multiple diseases